projects/fsastorefrontlib/src/cms-components/my-account/address-book/address-form/address-form.component.html

```diff
@@ -156,13 +154,20 @@
       </div>
     </div>
   </div>
-  <div class="cx-checkout-btns row">
-    <div class="col-md-12 col-lg-6" *ngIf="showCancelBtn">
-      <button class="btn btn-block btn-action" (click)="back()">
+  <div class="cx-checkout-btns row col-md-12 col-lg-12">
+    <div class="col-md-12 col-lg-6 pl-0" *ngIf="showCancelBtn">
+      <button class="btn btn-block btn-secondary mb-3" (click)="back()">
         {{ cancelBtnLabel || ('addressForm.chooseAddress' | cxTranslate) }}
       </button>
     </div>
-    <div class="col-md-12 col-lg-6">
+    <div
+      class="col-md-12 col-lg-6"
+      [ngClass]="
+        actionBtnLabel === ('addressBook.updateAddress' | cxTranslate)
+          ? 'pr-0'
+          : 'pl-0'
```

Comment:
Again overriding default col paddings?

Reply:
This won't be needed when I remove `col-md-12 col-lg-12` classes from `<div class="cx-checkout-btns row">`